docs/templates.md

```diff
@@ -95,6 +95,45 @@ The `coder_agent` resource can be configured as described in the
 For example, you can use the `env` property to set environment variables that will be
 inherited by all child processes of the agent, including SSH sessions.
 
+#### startup_script
+
+Use the Coder agent's `startup_script` to run additional commands like
+installing IDEs and clone dotfile and project repos. In this example, the
+project and dotfile repos are specified as Terraform input variables elsewhere
+in the `main.tf` Note the `&` after the `code-server` start to execute
+`code-server` process in the background so the `startup_script` can continue
+with the repo cloning steps.
+
+```hcl
+resource "coder_agent" "coder" {
+  os   = "linux"
+  arch = "amd64"
+  dir = "/home/coder"
+  startup_script = <<EOT
+#!/bin/bash
+
+# install code-server
+curl -fsSL https://code-server.dev/install.sh | sh 
+code-server --auth none --port &
```

Comment:
--port is left empty. this would return an error

```
error --port requires a value
```

Here's a fix

```suggestion
code-server --auth none --port 11337 &
```

Reply:
good catch, I accidentally deleted port with my 2>1& | tee stuff